.travis.yml

```diff
@@ -12,6 +12,8 @@ addons:
       - debian-sid
     packages:
       - shellcheck
+      - pwgen
+      - whois
```

Comment:
I'm on the fence actually... I was wondering whether this is surprising enough to be worth a comment here as well.